cryptofeed/exchange/upbit.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import json
+from decimal import Decimal
+import requests
+import zlib
```

Comment:
you appear to not be using zlib or base64

Reply:
Yor are right, I will remove the zlib and base64